executor/adapter_test.go

```diff
@@ -35,3 +41,40 @@ func (s *testSuiteP2) TestQueryTime(c *C) {
 	costTime = time.Since(tk.Se.GetSessionVars().StartTime)
 	c.Assert(costTime < 1*time.Second, IsTrue)
 }
+
+func (s *testSuiteP2) TestFormatSQL(c *C) {
+	preparedParams := variable.PreparedParams{
+		types.NewIntDatum(1),
+		types.NewFloat64Datum(2),
+		types.NewStringDatum("\"hello, 世界\""),
+		types.NewStringDatum("[1, 2, 3]"),
+		types.NewStringDatum("{}"),
+		types.NewStringDatum(`{"a": "9223372036854775809"}`),
+		mustParseTimeIntoDatum("2011-11-10 11:11:11.111111", mysql.TypeTimestamp, 6),
+	}
+	preparedSQL := executor.FormatSQL("select ?, ?, ?, ?, ?, ?, ?;", preparedParams)()
+	c.Check(preparedSQL, Equals, "select 1, 2, \"hello, 世界\", [1, 2, 3], {}, {\"a\": \"9223372036854775809\"}, 2011-11-10 11:11:11.111111;")
```

Comment:
I think string in select field should be quoted.
Right: `select "abc";`
Wrong: `select abc;`

Reply:
I will fix it.